### ISSUE TITLE
API 

### ISSUE THREAD

**Issue by Seluj78** [OpenApprenticeFoundation/openapprentice-website#36]
Add an `API` for openapprentice.

This might be a separate repository in the future.

hosted under `api.openapprentice.org`

**Comment by Seluj78** [comment]
Make it a RESTful API and GraphQl

**Comment by davidkartuzinski** [comment]
You read my mind! I believe it's important to be able to access the "student" and "mentor" as mentors or students but being able from different modules to "know who they are".  Do you need my input for any of this?

**Comment by Seluj78** [comment]
Yes Sir, I will need your input on this but the API is not the priority right now, as we have the website to be done first, I'll work on the API later ;)
